### PR TITLE
oculante: 0.7.7 -> 0.8.6

### LINKED ISSUE #275927: Update request: oculante 0.7.7 → 0.8.6
- Package name: oculante
- Latest released version: 0.8.6

- Current version on the unstable channel: 0.7.7
- Current version on the stable/release channel: 0.7.7

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**
@dit7ya @figsoda 

-----

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### LINKED ISSUE #276729: oculante: fails to start (missing opengl)
### Describe the bug

Fails to run due to error:
```
warning: queue 0x55f20471bb30 destroyed while proxies still attached:
  wl_output@14 still attached
  wl_seat@13 still attached
  xdg_activation_v1@12 still attached
  wp_viewporter@11 still attached
  zwp_text_input_manager_v3@10 still attached
  zwp_pointer_constraints_v1@9 still attached
  zwp_relative_pointer_manager_v1@8 still attached
  zxdg_decoration_manager_v1@7 still attached
  wl_subcompositor@6 still attached
  wl_compositor@5 still attached
  wl_shm@4 still attached
  wl_registry@2 still attached
Error: "Cannot select a valid OpenGL configuration: not found"
```

### Steps To Reproduce

```
nix run nixpkgs#oculante
```

### Expected behavior

It runs.

### Additional context
Running it with NixGL works. Using an Intel GPU.

### Notify maintainers

@figsoda @dit7ya

### Metadata

Running on NixOS 24.05pre, current nixpkgs revision is 6df37dc6a77654682fe9f071c62b4242b5342e04.

Bumping the version on master to 0.8.6 and building has the same result.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by t4ccer**
## Description of changes

Closes #275927 
Closes #276729

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by amesgen — APPROVED** [review]
Works great, fixes https://github.com/NixOS/nixpkgs/issues/276729 for me  :+1:

**Review by geri1701 — APPROVED** [review]
fixes https://github.com/NixOS/nixpkgs/issues/276729

File: pkgs/applications/graphics/oculante/default.nix [hunk 1]
```diff
@@ -69,10 +67,8 @@ rustPlatform.buildRustPackage rec {
   postInstall = ''
     install -Dm444 $src/res/oculante.png -t $out/share/icons/hicolor/128x128/apps/
     install -Dm444 $src/res/oculante.desktop -t $out/share/applications
-  '';
-
-  postFixup = lib.optionalString stdenv.isLinux ''
-    patchelf $out/bin/oculante --add-rpath ${lib.makeLibraryPath [ libxkbcommon libX11 ]}
+    wrapProgram $out/bin/oculante \
```
Comment (figsoda) [inline pkgs/applications/graphics/oculante/default.nix:70]: What's the reason for this change?

Comment (geri1701) [reply pkgs/applications/graphics/oculante/default.nix:70]: I think this: https://github.com/NixOS/nixpkgs/issues/276729

Comment (t4ccer) [reply pkgs/applications/graphics/oculante/default.nix:70]: I doubt it ever worked honestly, if it did then by luck of having some env vars set up properly via desktop environment. Even on 2e59fd5ffc1a15f5a71d1593a8969f9608edca0b I get libgl errors on startup

**Review by figsoda — APPROVED** [review]
